lib/Byng/Pimcore/Elasticsearch/Query/MatchQuery.php

```diff
@@ -48,21 +49,23 @@
      * @param string|array $query
      * @param string       $operator
      */
-    public function __construct($field, $query, $operator = self::OPERATOR_AND)
+    public function __construct($field, $query, $operator = null)
     {
         $this->field = $field;
         $this->query = $query;
         $this->operator = $operator;
 
-        switch ($this->operator) {
-            case self::OPERATOR_AND:
-            case self::OPERATOR_OR:
-                break;
-            default:
-                throw new \InvalidArgumentException(sprintf(
-                    "Unexpected operator found: '%s'",
-                    $this->operator
-                ));
+        if ($operator !== null) {
```

Comment:
I think a simple match query will be more common and also when searching on a specific field then this can be a little too restrictive but it's there if required.